keepassxc-browser/options/options.html

```diff
@@ -29,9 +29,9 @@ <h3 class="muted"><img src="/icons/keepassxc_48x48.png" alt="logo" /> KeePassXC-
         <h2>General Settings</h2>
         <hr />
         <p>
-          If you just want to insert username + password into the fields where your focus is, press <code>Alt + Shift + U</code>.
+          If you just want to insert username + password into the fields where your focus is, press <code><span id="mac-user-shortcut">Ctrl + Shift + U</span><span id="default-user-shortcut">Alt + Shift + U</span></code>.
```

Comment:
Doesn't Mac use "Cmd" instead of Ctrl?

Reply:
Using Cmd with macOS can trigger browser's own shortcuts. In this case Ctrl is actually MacCtrl.